### PR TITLE
21 seccion servicios

### PR DESCRIPTION
**Pull request by Roxha-V**
Improve text alignment and elements responsiveness, specifically in the animation (hover) that is generated from the user interaction.
    
    I would need to check it and be able to merge it with develop.
    
    There would be one last change in style regarding how the service detail property of element [0] of carData is displayed. To evaluate .

### CODE REVIEW

File: app/components/services-section/services-card/index.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+'use client'
+
+import { icon } from '@fortawesome/fontawesome-svg-core';
+import React, { useEffect } from 'react';
+
+
+function Card({ id, title, img, bgColor, service_detail }) {
+
```
Comment (devPalter) [inline app/components/services-section/services-card/index.jsx]: unused comments

File: app/components/services-section/services-card/index.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    // Recordar : El operador lógico AND (&&) será true para un conjunto de operandos booleanos,solo si todos los operandos son true.
+    //         sino sera false.
+
+
+    // aca se entiende que isOpen es la card cardAbierta pero que en su estado inicial es false (osea que no esta isOpen),
+    // mientras que si se usa el operador ! este estado cambia, osea es el opuesto
+
+
+
+    useEffect(() => {
```
Comment (devPalter) [inline app/components/services-section/services-card/index.jsx]: which is the propousal of this implementation?

File: app/components/services-section/services-card/index.jsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
+'use client'
+
+import { icon } from '@fortawesome/fontawesome-svg-core';
```
Comment (devPalter) [inline app/components/services-section/services-card/index.jsx]: this import is not working and is unsed, we should remove it?

File: app/components/services-section/services-card/index.jsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+            };
+        }
+    }, []); // Se ejecuta solo una vez después de que el componente se monta
+
+
+
+    return (
+
+
+        <div id="touch-card" className="group relative cursor-pointer overflow-hidden bg-white pt-10 pb-8 shadow-xl ring-1 ring-gray-900/5 transition-all duration-300 hover:-translate-y-1 hover:shadow-2xl sm:mx-auto rounded-lg  px-6 sm:px-10">
```
Comment (devPalter) [inline app/components/services-section/services-card/index.jsx]: todas las cards generadas tendran el mismo id esto es correcto?

**Review by devPalter — CHANGES_REQUESTED** [review]

File: app/components/services-section/services-card/index.jsx (hunk 2 above)
Comment (Roxha-V) [reply app/components/services-section/services-card/index.jsx]: se intenta emular el hover de css para dispositivos tactiles

Comment (devPalter) [reply app/components/services-section/services-card/index.jsx]: excelente! lo dejemos para otra tarea dado que es dificil de emular su funcionamiento y el codigo pareciera no hacer lo esperado

**Review by devPalter — APPROVED** [review]

**Pull request merged**